fs/nfs/Kconfig

```
@@ -13,6 +13,15 @@ config NFS
 
 #if NFS
 
+config NFS_DONT_BIND_TCP_SOCKET
```

```suggestion
config NFS_NO_BIND_TCP_SOCKET
```
Not sure if this is better. I'm ok to keep existing name

---

@pkarashchenko 
Let me keep the name